scripts/uri-input.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+import logging
+import mimetypes
+import os
+import string
+import subprocess
+from enum import Enum
+from pathlib import Path
+from typing import Any, Dict, Optional
+from urllib.parse import urlparse
+
+import click
+
+from savant.utils.file_types import FileType, parse_mime_types
+
+logger = logging.getLogger('uri-input')
+
+adapters_dir = Path(__file__).parent.parent / 'adapters' / 'gst' / 'sources'
```

Comment:
Let's rename the `adapters_dir` variable using uppercase, to distinguish from local variables.